foster55f/market2table-fe
# VendorSelector Component for vendors to find their account or create one

### What does this PR do?
Creates VendorsSelectContainer:
- creates form to search for vendors by name, renders cards and Create New Vendor button
- uses local state to control search form - when search is clicked it filters all
Vendors by name to match what is in search field. stores filtered items in localstore to display. When clear is clicked it Displays all vendors again.
- Contains large button to create a new vendor routing to the VendorForm
- created VendorCards via vendors that have been filtered

Creates VendorCard:
- displays the vendor Name
- button that routes to MarketLinkForm 
- button that routes to VendorForm and performs GET request to get Vendor 
products and combines that with vendor info and puts it in the Store under 
SelectedVendor

 - dynamically change vendor login button to a home button when on vendor pages 
 - home button takes you back to landing page '/'

### How to test?
 - Pull down branch
 - run 'npm start' and the site should open in browser via localhost:3000
 - Click the vendor login button to see component 
 - Fill out input and hit search or 'enter' and then 'clear'

### Issues:
No issues


### Relevant Screenshots:
<img width="1426" alt="Screen Shot 2020-02-20 at 3 42 14 PM" src="https://user-images.githubusercontent.com/49846853/74987376-49994e00-53f8-11ea-9542-637057d3d26b.png">

<img width="1426" alt="Screen Shot 2020-02-20 at 3 42 26 PM" src="https://user-images.githubusercontent.com/49846853/74987382-4d2cd500-53f8-11ea-9047-bec02067e709.png">

<img width="1426" alt="Screen Shot 2020-02-20 at 3 42 32 PM" src="https://user-images.githubusercontent.com/49846853/74987398-54ec7980-53f8-11ea-9e9a-e0e85bddbb80.png">


Thanks!
@foster55f
